README.md

```diff
@@ -60,57 +60,20 @@ Download the pre-compiled package for your operating system.  Install required d
   If the imaging install script reports errors in creating directories (due to /data/ mount permissions), review and manually execute `mkdir/chmod/chown` commands starting at [imaging_install.sh:L97](https://github.com/aces/Loris-MRI/blob/master/imaging_install.sh#L97)
 
   Note: The installer will allow Apache to write to the /data/ directories by adding user lorisadmin to the Apache linux group.  To ensure this change takes effect, log out and log back into your terminal session before running the imaging pipeline.
-The installer will also set Apache group ownership of certain /data/ subdirectories.  
+The installer will also set Apache group ownership of certain /data/ subdirectories.
 
-#### 4. Configure paths and environment
-
-  To help ensure Apache-writability, verify that your environment file contains the following line:
+  After having run the install script, it is possible to verify that certain
+    fields were correctly populated by `imaging_install.sh` (see
+    [section 2.4](02-Install.md#2.4-post-installation-checks)).
```

Comment:
@MounaSafiHarab Actually, I did put the link but right after having finished running the install script (at the end of section 3 of the README).  It felt less weird putting it there than at the end. What do you think?